app/api.py

```diff
@@ -124,6 +126,12 @@ async def authentication(form_data: Annotated[OAuth2PasswordRequestForm, Depends
     raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Server error")
 
 
+@app.get("/prices", response_model=List[schemas.PriceBase])
```

Comment:
we can use directly list[schemas.PriceBase] instead of List, as we're running on Python 3.10